WooCommerce/WooCommerceUITests/README.md

```diff
@@ -72,25 +72,25 @@ Note that due to the mock server setup, tests cannot be run on physical devices
 
 You can run the tests locally with these steps:
 
-1. Follow the [build instructions](../README.md#build-instructions) to clone the project, install the dependencies, and open the project in Xcode.
+1. Follow the [build instructions](../../README.md#build-instructions) to clone the project, install the dependencies, and open the project in Xcode.
 2. Run `rake mocks` to start a local mock server.
-3. With the `WooCommerce` scheme selected in Xcode, navigate to Product > Test Plan and select `UITests`, or open the Test Navigator and select the `UITests` test plan.
+3. Select the `WooCommerceUITests` scheme in Xcode.
```

Comment:
I'd suggest leaving most of the original wording. It is clearer about how to select the correct scheme.

Reply:
The reason why I went this way, is because `WooCommerce` scheme has `Unit Tests` selected as a default test plan, so to run the UI tests while having `WooCommerce` scheme selected would require editing the scheme every time.

`WooCommerceUITests` scheme is a dedicated scheme which by default has `UI Tests` test plan activated - so it's a matter of one action VS three actions.